<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceStatusForm</class>
 <widget class="QWidget" name="DeviceStatusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
border-image: url(:/images/down_gre_dou.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="DeviceButton" name="pushButton_dev2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Dev2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_node2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/sw_gre_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_link8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/up_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="DeviceButton" name="pushButton_dev4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dev4</string>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/down_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/up_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_node1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/sw_gre_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="DeviceButton" name="pushButton_dev1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Dev1</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="DeviceButton" name="pushButton_dev6">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dev6</string>
    </property>
   </widget>
   <widget class="DeviceButton" name="pushButton_dev3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dev3</string>
    </property>
   </widget>
   <widget class="DeviceButton" name="pushButton_dev5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dev5</string>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/down_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/up_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/down_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/down_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LinkButton" name="Button_link4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/up_gre_dou_alp.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_dev_info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid; border-radius: 5px;\n</string>
    </property>
    <property name="text">
     <string>设备状态信息：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeviceButton</class>
   <extends>QPushButton</extends>
   <header>devicebutton.h</header>
  </customwidget>
  <customwidget>
   <class>LinkButton</class>
   <extends>QPushButton</extends>
   <header>linkbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
